src/nomad_material_processing/combinatorial.py

```diff
@@ -363,4 +365,251 @@ def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
         super().normalize(archive, logger)
 
 
+class CombinatorialProperty(ArchiveSection):
+    model = Quantity(
+        type=str,
+        description="""
+        The model used to calculate the property.
+        """,
+    )
+    measurements = Quantity(
+        type=Reference(Measurement.m_def),
+        shape=['*'],
+    )
+
+
+class Formula(CombinatorialProperty):
+    value = Quantity(
+        type=str,
+        description="""
+        The formula of the sample.
+        """,
+    )
+
+
+class Thickness(CombinatorialProperty):
+    value = Quantity(
+        type=float,
+        description="""
+        The (average) thickness of the sample.
+        """,
+        unit='m',
+    )
+
+
+class Conductivity(CombinatorialProperty):
+    value = Quantity(
+        type=float,
+        description="""
+        The conductivity of the sample.
+        """,
+        unit='S/m',
+    )
+
+
+class CarrierLifetime(CombinatorialProperty):
+    value = Quantity(
+        type=float,
+        description="""
+        The lifetime of the (majority) carriers in the sample.
+        """,
+        unit='s',
+    )
+
+
+class BandGap(CombinatorialProperty):
+    value = Quantity(
+        type=float,
+        description="""
+        The band gap of the sample.
+        """,
+        unit='eV',
+    )
+
+
+class Synthesis(CombinatorialProperty):
+    method = Quantity(
+        type=str,
+        description="""
+        The method used to synthesize the material.
+        """,
+    )
+    temperature = Quantity(
+        type=float,
+        description="""
+        The (maximum) temperature at which the material was synthesized.
+        """,
+        unit='K',
+    )
+    pressure = Quantity(
+        type=float,
+        description="""
+        The (average) pressure at which the material was synthesized.
+        """,
+        unit='Pa',
+    )
+    atmosphere = Quantity(
+        type=str,
+        description="""
+        The atmosphere in which the material was synthesized.
+        """,
+    )
+
+
+class Photoluminescence(CombinatorialProperty):
+    peak_position = Quantity(
+        type=float,
+        description="""
+        The peak position of the photoluminescence spectrum.
+        """,
+        unit='nm',
+    )
+    fwhm = Quantity(
+        type=float,
+        description="""
+        The full width at half maximum of the photoluminescence spectrum.
+        """,
+        unit='nm',
+    )
+    peak_area = Quantity(
+        type=float,
+        description="""
+        The peak area of the photoluminescence spectrum.
+        """,
```

Comment:
Shouldn't units be defined for peak_area since we have quantities with units on at least x-axis (wavelength, or energy in general, I assume)?

Reply:
not sure what the correct unit is, if it is wavelength over intensity, i think leaving it like this is ok